packages/node/src/BacktraceClient.ts

```diff
@@ -219,7 +221,7 @@ export class BacktraceClient extends BacktraceCoreClient {
     }
 
     private async loadNodeCrashes() {
-        if (!this.options.database?.captureNativeCrashes) {
+        if (!this.fileSystem || !this.options.database?.captureNativeCrashes) {
```

Comment:
this is out of scope for this review, however I have a question:

- client is a programming interface for developer to use Backtrace. For managing database we have a separated class which is BacktraceDatabase. Shouldn't this code be a part of the database somehow?

Reply:
I think not, because depending all stuff on BacktraceDatabase that can be dependent just on FileSystem is wrong. It's completely unnecessary.

It makes sense for me that FileSystem is the dependency of BacktraceDatabase and other stuff that may use it, and not have the DB inbetween.
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/6e0446bb-ea4d-4409-b66e-6160f11c5086)

Reply:
but here you have something that clearly depends on the database. You literally need to check if the database is initialized and do something only when it is.

Reply:
that's a business decision, not a code decision.